samples/algorithms/QuantumHelloWorld.qs

```diff
@@ -0,0 +1,24 @@
+/// # Quantum Hello World!
+///
+/// This Q# code generates a random bit by setting a qubit in a superposition of the
+/// computational basis states |0〉 and |1〉, and returning the measurement result.
+namespace QuantumHelloWorld {
+    // Import the `Message` function and the `H` operation used in the program.
```

Comment:
If we keep the import, the comment would benefit from additional info, since this is also where `M` and `Reset` come from.

Reply:
Ended up removing the `open` statement based on a previous suggestion.